Makefile

```
@@ -34,6 +34,7 @@ test-online:
 checkstyle: test-shellcheck test-yaml
 
 test-shellcheck:
+	shfmt -d .
```

I just spotted this, it is only a suggestion for consistency, feel free to ignore that.

```suggestion
    @which shfmt >/dev/null 2>&1 || echo "Command 'shfmt' not found, can not execute shell script formating checks"
	shfmt -d .
```

---

We should also probably add the package to our dependencies: https://github.com/os-autoinst/scripts/blob/master/dependencies.yaml
Maybe add a new target `develop`